### PR TITLE
Add circleci badge

### PR DESCRIPTION
**Pull request by AndreaCrotti**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ Jackdaw is a Clojure library for the Apache Kafka distributed streaming platform
 
 To use the latest release, add the following to your project:
 
+[![CircleCI](https://circleci.com/gh/FundingCircle/jackdaw.svg?style=svg)](https://circleci.com/gh/FundingCircle/jackdaw)
```
Comment (cddr) [inline README.md]: The list of badges looks a bit weird when prefixed with the sentence "To use the latest release, add the following to your project". Maybe we should split out the clojars one from the others?

screen shot 2019-02-25 at 9 24 43 pm

Comment (AndreaCrotti) [reply README.md]: Yes that's a good point, well I just suggest to move them all on top and remove the extra docs, I think Clojure devs can recognize the clojars badge anyway we don't really need to specify it

File: README.md [hunk 2]
```diff
@@ -1,16 +1,12 @@
...
-
-Jackdaw is a Clojure library for the Apache Kafka distributed streaming platform. With Jackdaw, you can create and list topics using the AdminClient API, produce and consume records using the Producer and Consumer APIs, and create stream processing applications using the Streams API. Jackdaw also contains functions to serialize and deserialize records as JSON, EDN, and Avro, as well as functions for writing unit and integration tests.
-
-
-## Installation
-
-To use the latest release, add the following to your project:
-
 [![Clojars Project](https://img.shields.io/clojars/v/fundingcircle/jackdaw.svg)](https://clojars.org/fundingcircle/jackdaw)
+[![CircleCI](https://circleci.com/gh/FundingCircle/jackdaw.svg?style=svg)](https://circleci.com/gh/FundingCircle/jackdaw)
```
Comment (creese) [inline README.md]: Let's move the CircleCI badge to the far right and find one that matches. We may also want to replace the graphic with text while we rethink the layout.

Example:
https://github.com/facebook/react

**Comment by AndreaCrotti** [comment]
Ok @creese with the latest changes it looks like this (which I think it's pretty similar to the react example):
jackdaw_badges

**Pull request merged**